### ISSUE TITLE
Move to v3 API

### ISSUE THREAD

**Issue by avram** [avram/zandy#138]
Move to V3 API and its support for better syncing and partial syncs

**Comment by judeibe** [comment]
Been looking at API 3, it supports JSON can the app implementation use that.

**Comment by avram** [comment]
Yes, it would be great to eliminate the XML handling components of the sync system-- that is an unpleasant and complex pile of code.

**Comment by judeibe** [comment]
I have started working on the code. I will create a pull request. I am using gson and  retrofit.
